mainnet.env

```diff
@@ -0,0 +1,114 @@
+# Network to run the node on ("mainnet","alfajores" or "baklava")
+NETWORK_NAME=mainnet
+
+# Type of node to run ("full" or "archive"), note that "archive" is 10x bigger
+NODE_TYPE=full
+
+###############################################################################
+#                            ↓ REQUIRED (BEDROCK) ↓                           #
+###############################################################################
+
+# L1 node that the op-node (Bedrock) will get chain data from
+OP_NODE__RPC_ENDPOINT=https://ethereum-rpc.publicnode.com
+
+# L1 beacon endpoint, you can setup your own or use Quicknode
+OP_NODE__L1_BEACON=https://ethereum-beacon-api.publicnode.com
+
+# Type of RPC that op-node is connected to, see README
+OP_NODE__RPC_TYPE=basic
+
+# Reference L2 node to run healthcheck against
+HEALTHCHECK__REFERENCE_RPC_PROVIDER=https://forno.celo.org
+
+###############################################################################
+#                            ↓ OPTIONAL (BEDROCK) ↓                           #
+###############################################################################
+
+# Optional path to a datadir for an L1 node to serve RPC requests requiring historical states. If
+# set an L1 node will be run in archive mode to serve requests requiring state for blocks prior to the
+# L2 hardfork and op-geth will be configured to proxy those requests to the L1 node.
+HISTORICAL_RPC_DATADIR_PATH=
+
+# Optional provider to serve RPC requests requiring historical state, if set op-geth will proxy
+# requests requiring state prior to the L2 start to here. If set this overrides the use of a local L1
+# node via HISTORICAL_RPC_DATADIR_PATH.
+OP_GETH__HISTORICAL_RPC=
+
+# Set to "full" to force op-geth to use --syncmode=full
+OP_GETH__SYNCMODE=
+
+IS_CUSTOM_CHAIN=true
+
+# Path to the datadir, If the datadir is empty then a new datadir will be
+# initialised at the given path.
+#
+# The path can be absolute, or relative to the docker-compose.yml file.
+DATADIR_PATH=./envs/${NETWORK_NAME}/datadir
+
+# If the datadir is on a disk that doesn't support unix domain sockets then you
+# will need to specify a path to a disk that does support unix domain sockets.
+# E.g. your normal hard drive. If left unset the default path inside the
+# datadir will be used.
+IPC_PATH=
+
+###############################################################################
+#                            ↓ REQUIRED (EIGENDA) ↓                           #
+###############################################################################
+
+# Specifies the endpoint of the eigenda proxy to use. If this is unset then a local eigenda proxy will be used.
+EIGENDA_PROXY_ENDPOINT=
+
+# Set to 0 if EIGENDA_PROXY_ENDPOINT is defined otherwise empty. This shouldn't be set manually.
+USE_LOCAL_EIGENDA_PROXY_IF_UNSET=${EIGENDA_PROXY_ENDPOINT:+0}
+
+EIGENDA_LOCAL_DISPERSER_RPC=disperser.eigenda.xyz:443 
+EIGENDA_LOCAL_SVC_MANAGER_ADDR=0x870679e138bcdf293b7ff14dd44b70fc97e12fc0
```

Comment:
@gastonponti could you verify if these are correct for mainnet?

Reply:
Not sure what the official source of truth is, but https://github.com/Layr-Labs/eigenlayer-middleware/blob/a7732a90f88090e75a1bed118b8db671868dda15/README.md?plain=1#L84 shows `0x870679e138bcdf293b7ff14dd44b70fc97e12fc0`

Reply:
Thanks @karlb!